package.json

```diff
@@ -90,7 +90,7 @@
   },
   "dependencies": {
     "lodash": "^4.17.4",
-    "patternfly-react": "^1.8.0",
+    "patternfly-react": "^1.16.4",
```

Comment:
might make sense to bump to 2.x as it already introduced api changes (for something we dont use yet - but can't see the value of using an older version now).

Reply:
I need this compatible with ui-classic. The goal is to replace textualsummaries in ui-classic.